themes/bootstrap5/templates/hierarchy/tree.phtml

```diff
@@ -32,12 +32,16 @@
     foreach ($nodes as $node) {
       $hasChildren = !empty($node->children);
       $liClasses = [];
+      $openTopLevel = false;
       if ($hasChildren) {
         $liClasses[] = 'hierarchy-tree__parent';
       }
       if ('collection' === $node->type) {
         $liClasses[] = 'hierarchy-tree__collection';
         $icon = 'hierarchy-collection';
+        if ($nodeId === 0 && !$parentNodeId && count($nodes) === 1) {
```

Comment:
Would it be a good idea to set `$nodeCount = count($nodes);` above the foreach loop and then use the variable here, so if there are a large number of nodes to check, we only need to perform the count operation a single time?